python/LAMBDA_CONCURRENCY_CHECK/LAMBDA_CONCURRENCY_CHECK.py

```diff
@@ -0,0 +1,476 @@
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+
+    concurrency = configuration_item['supplementaryConfiguration']['Concurrency']['reservedConcurrentExecutions']
+
+    if 'ConcurrencyLimitLow' in valid_rule_parameters and 'ConcurrencyLimitHigh' not in valid_rule_parameters:
+        if concurrency < valid_rule_parameters['ConcurrencyLimitLow']:
+            return build_evaluation_from_config_item(
+                configuration_item,
+                'NON_COMPLIANT',
+                annotation='Concurrency of AWS Lambda function {} is lower then {}.'.format(
+                    function_name,
+                    valid_rule_parameters['ConcurrencyLimitLow']
+                    )
+                )
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+
+    if 'ConcurrencyLimitHigh' in valid_rule_parameters and 'ConcurrencyLimitLow' not in valid_rule_parameters:
+        if concurrency > valid_rule_parameters['ConcurrencyLimitHigh']:
+            return build_evaluation_from_config_item(
+                configuration_item,
+                'NON_COMPLIANT',
+                annotation='Concurrency of AWS Lambda function {} is higher then {}.'.format(
+                    function_name,
+                    valid_rule_parameters['ConcurrencyLimitHigh']
+                    )
+                )
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+
+    if valid_rule_parameters['ConcurrencyLimitLow'] <= concurrency <= valid_rule_parameters['ConcurrencyLimitHigh']:
+        return build_evaluation_from_config_item(configuration_item, 'COMPLIANT')
+    return build_evaluation_from_config_item(
+        configuration_item,
+        'NON_COMPLIANT',
+        annotation='AWS Lambda function {} concurrency is not within bounds of {} and {}.'.format(
+            function_name,
+            valid_rule_parameters['ConcurrencyLimitLow'],
+            valid_rule_parameters['ConcurrencyLimitHigh']
+            )
+        )
+
+# Check if parameters are defined and has values > 0
+def evaluate_parameters(rule_parameters):
+    if not rule_parameters:
+        return {}
+    if 'ConcurrencyLimitLow' in rule_parameters:
+        if int(rule_parameters['ConcurrencyLimitLow']) <= 0:
+            raise ValueError('ConcurrencyLimitLow must be a positive integer greater than 0.')
+        rule_parameters['ConcurrencyLimitLow'] = int(rule_parameters['ConcurrencyLimitLow'])
+    if 'ConcurrencyLimitHigh' in rule_parameters:
+        if int(rule_parameters['ConcurrencyLimitHigh']) <= 0:
+            raise ValueError('ConcurrencyLimitHigh must be a positive integer greater than 0.')
+        rule_parameters['ConcurrencyLimitHigh'] = int(rule_parameters['ConcurrencyLimitHigh'])
+    if all(key in rule_parameters for key in ['ConcurrencyLimitLow', 'ConcurrencyLimitHigh']) and \
+        rule_parameters['ConcurrencyLimitLow'] >= rule_parameters['ConcurrencyLimitHigh']:

```

Comment:
Shoud be `>` not `>=`. If it is equal it mandates a specific value.

Reply:
Done!